pve-iso-2-pxe.sh

```diff
@@ -57,8 +60,8 @@ case "${mimetype##*/}" in
 esac
 $decompress || exit 4
 echo "adding iso file ..." 
-echo "../proxmox.iso" | cpio -L -H newc -o >> /tmp/initrd. || exit 5
-mv /tmp/initrd. initrd
+echo "../proxmox.iso" | cpio -L -H newc -o >> initrd || exit 5
```

Comment:
Does not work for gzipped (6) versions.

`gzip -S img -d /tmp/initrd.img` will extract the `initrd.img` into `initrd.` (due to the suffix)